VagrantProvision.sh

```diff
@@ -244,21 +334,22 @@ PATH=$HOOT_HOME/bin:$PATH
 EOT
 fi
 
-if ! grep -i --quiet 'umask 002' /etc/default/tomcat6; then
-echo "Changing Tomcat umask to group write..."
+# Was 002, changing it to 000 to get rid of errors
+if ! grep -i --quiet 'umask 000' /etc/default/tomcat6; then
+echo "### Changing Tomcat umask to group write..."
 sudo bash -c "cat >> /etc/default/tomcat6" <<EOT
 # Set tomcat6 umask to group write because all files in shared folder are owned by vagrant
-umask 002
+umask 000
```

Comment:
I don't know.  Thinking about access, I'll have a go adding tomcat6 to the vagrant group and see if that works.

If you use the Vagrantfile that is in the branch, the hoot directory gets rsynce'd into the VM and there are no problems with chown etc. The files are all inside the VM and things run well. 

If we use nfs on Selma then we have problems with trying to use chown etc to modify file access. We have this situation:
* Host files are jenkins:jenkins  (host uid 110, host gid 120)
* Inside the vm: vagrant:vagrant (uid 1000,gid 1000)
* Inside the vm: tomcat6 (uid 106, git 116)

We are nfs mounting the files and then using bindfs to "bend" the user and groups so we can read/write them.
From VagrantfileLocal:
```
      mkdir -p /home/vagrant/hoot
      mkdir -p /home/vagrant/.hoot-nfs

      sudo mount XXXXXXXXXX:#{Dir.pwd} /home/vagrant/.hoot-nfs

      # Map users: 110 (host Jenkins) to Vagrant and tomcat6 to Vagrant
      sudo bindfs --map=110/vagrant:vagrant/tomcat6:@120/@vagrant:@vagrant/@tomcat6 /home/vagrant/.hoot-nfs /home/vagrant/hoot
```